packages/odyssey-react/src/components/Radio/RadioGroup/index.tsx

```diff
@@ -72,7 +71,7 @@ export interface Props
  * Radios appear as a ring shaped UI accompanied by a caption that allows
  * the user to choose only one option at a time.
  */
-const RadioGroup: FunctionComponent<Props> = (props) => {
+const RadioGroup = (props: Props): JSX.Element => {
```

Comment:
in the future we can look at improving this return type even further via the `ReactElement` generic.